akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -314,7 +314,7 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
    * a certain size.
    */
   def registerOnMemberUp[T](code: ⇒ T): Unit =
```

Comment:
There were some cases where type inference did not work with `Unit`, while this signature says “give me whatever”.